packages/react-jsx-highcharts/src/utils/getBoundChartMethod.js

```diff
@@ -0,0 +1,11 @@
+export default function getBoundChartMethod(chart, method, context) {
+  const boundMethod = method.bind(context);
+
+  return function (...rest) {
+    if (!chart.__destroyed) {
+      return boundMethod(...rest)
+    }
+
+    return null;
```

Comment:
As I mentioned above, this construction isn't due to `return null` here. The `noop` is a nice idea though, so `getBoundChartMethod` now encapsulates `context && context.method` part from providers.